simulation-runner/run_sbt.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Missed this comment, sorry. Yeah that sounds reasonable to me, I don't have a strong opinion